tests/semantic_analyser.cpp

```diff
@@ -2597,9 +2608,7 @@ BEGIN { (long)cpu }
 stdin:1:9-15: ERROR: Cannot cast to "long"
 BEGIN { (long)cpu }
         ~~~~~~
-stdin:1:9-15: HINT: Did you mean "int64"?
-BEGIN { (long)cpu }
```

Comment:
Love that this isn't repeated anymore

Reply:
This should be the ~only functional change here.